### PR TITLE
[JP] Improve Enum basics section

### PR DESCRIPTION
**Pull request by hinamiyagk**
#534

I can't understand meaning of 'combining enum with documentation' enough. What do you mean?

### CODE REVIEW

File: jp/lessons/basics/enum.md [hunk 1]
```diff
@@ -12,9 +12,30 @@ lang: jp
...
+any?/1
+any?/2
+at/2
+at/3
+...
+```
+
+これを見れば`Enum`モジュールに大量の機能があるのが明らかで、これには明確な理由があります。
+リストなどの列挙は関数型プログラミングの核で、信じられないほど有用です。
+おまけにリストなどの列挙は、以前見たような言語レベルでサポートされているドキュメントといった、Elixirの他の要素と組み合わせて活用することで、信じられないような効果をもたらします。
```
Comment (marocchino) [inline jp/lessons/basics/enum.md]: "組み合わせて活用することで"よりは"共に"に近いと思います。

**Comment by doomspork** [comment]
@3100 can you take a peek at this when you get a chance?

**Comment by doomspork** [comment]
@hinamiyagk are you referring to this line: "By leveraging it combined with other perks of Elixir, such as documentation being a first class citizen as we just saw, it can be incredibly empowering to the developer as well."

**Comment by doomspork** [comment]
@hinamiyagk / @riseshia / @marocchino — can this be merged?

**Comment by riseshia** [comment]
I think it could, however, it will be better if review applied. how do you think @marocchino ?

**Comment by marocchino** [comment]
I think it's a negligible difference.

retranslate original will be:
> Besides, enumerating lists etc., combined with other elements of Elixir, such as documents supported at the language level as we saw earlier, will give incredible effects.

my suggestion was:
> Besides, enumerating lists etc. brings an incredible effect by using Elixir together, such as documents supported at language level as we saw before.

**Comment by doomspork** [comment]
Howdy @hinamiyagk / @marocchino / @riseshia — can this be merged?

**Comment by riseshia** [comment]
It could be, proposal from @marocchino could be another pr. XD

**Comment by doomspork** [comment]
Okay, thank you @riseshia!

**Comment by riseshia** [comment]
@doomspork how handle above suggestion?

**Comment by doomspork** [comment]
@riseshia if you or @marocchino want to open a PR for the change that'd be great!

**Pull request merged**